esbuild.config.mjs

```diff
@@ -20,9 +20,10 @@ esbuild.build({
 	external: ['obsidian', 'electron', ...builtins],
 	format: 'cjs',
 	watch: !prod,
-	target: 'es2016',
+	target: 'es2018',
 	logLevel: "info",
 	sourcemap: prod ? false : 'inline',
 	treeShaking: true,
+	minify: prod,
```

Comment:
Does this minify the main.ja?

Reply:
It minifies it for production, but not dev. You assign the variable above.

Reply:
This gets imported in the esbuild file, but isn't in the dev dependencies. It doesn't have any real impact, it just makes it more explicit.